glossary-sync.py

```diff
@@ -188,26 +177,32 @@ def _glossary_term_from_datahub(
 def fetch_datahub_glossary():
     graph = get_graph()
 
-    # Get all the urns in the glossary.
+    logger.info("Get all the urns in the glossary.")
     urns = list(graph.get_urns_by_filter(entity_types=["glossaryTerm", "glossaryNode"]))
+    logger.info(f"Got {len(urns)} urns")
 
-    # Hydrate them into entities.
-    entities = {urn: graph.get_entity_semityped(urn) for urn in urns}
+    logger.info("Hydrate them into entities.")
+    entities = {}
+    for urn in progressbar.progressbar(urns):
```

Comment:
Why don't we collect glossaryNodes and glossaryTerm in a different dict? 
Something like:
```suggestion
    glossary_node_enitites = {}
    glossary_term_entities = {}
    for urn in progressbar.progressbar(urns):
       try:
             entity = graph.get_entity_semityped(urn)
             if guess_entity_type(urn) != "glossaryNode":
                 glossary_node_enitites[urn] = entity
             elif guess_entity_type(urn) != "glossaryTerm": 	   				    
                 glossary_term_enitites[urn] = entity
...		          		          
```
Later in the code, you don't have to check for each entity if it is glossaryTerm or glossaryNode